testing/simapp/app.go

```diff
@@ -107,7 +107,7 @@ import (
 	ibcclientclient "github.com/cosmos/ibc-go/v6/modules/core/02-client/client"
 	ibcclienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
 	porttypes "github.com/cosmos/ibc-go/v6/modules/core/05-port/types"
-	ibchost "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+	ibcexported "github.com/cosmos/ibc-go/v6/modules/core/exported"
```

Comment:
We should document this change in the migration docs for v7, right? And also update the usages of `ibchost` [in  `integration.md`](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/integration.md).